components/component.css

```diff
@@ -30,7 +31,7 @@
     --light-grey-color:#666;
     --complementory-color:#CAF36C;
     --grey-dark: gray;
```

Comment:
you can use hex code here as other colors are also in hex-code

Reply:
OK will do